### ISSUE TITLE
BUG: No icon for `app.postcss` (v3.1.0)

### ISSUE THREAD

**Issue by babakfp** [material-theme/vsc-material-theme-icons#229]
It seems like in this [commit](https://github.com/material-theme/vsc-material-theme-icons/commit/4e61af7b95012b2475b79cd1a6d5944c8263368a) this code `"postcss": "_file_postcss",` got removed and because of that this file doesn't have a custom icon anymore.

![image](https://github.com/material-theme/vsc-material-theme-icons/assets/44144724/aa61c31f-0857-4d88-9b80-17fdec1dd67e)

**Comment by equinusocio** [comment]
Hello, the postcss extension is not recommended, as the `.pcss` is the one suggested. 

Back to 2015:
https://x.com/postcss/status/661645290622083073?s=46

We can restore it but you should definitely consider to rename all `.postcss` files into `.pcss`

**Comment by babakfp** [comment]
Hi

Hmm

Good to know!

I dug into this a little bit and this is what I found ([vsCode PostCSS Extension](https://marketplace.visualstudio.com/items?itemName=csstools.postcss#:~:text=PostCSS%20Language%20Support%20adds%20support,pcss%20and%20.)):

> This extension is applied to .css, .pcss and .postcss files.

TailwindCSS just uses the `.css` extension.

I got familiar with the `.postcss` extension because it was been used in [this project](https://github.com/svelte-add/tailwindcss). It created a file named `app.postcss` for Svelte projects.

I think it's a good idea to support the `.postcss` extension since it's supported by the PostCSS vsCode Extension and it's been used in some projects and some people same as I use the `.postcss` extension.

PostCSS config files contain the word "postcss". So, I think I'm going to stick to the `.postcss` extension, also, it looked alien to me when I renamed it to `.pcss` 🤷‍♂️.
